source/common/http/conn_manager_impl.cc

```diff
@@ -708,12 +708,19 @@ void ConnectionManagerImpl::ActiveStream::decodeHeaders(HeaderMapPtr&& headers,
     stream_info_.setDownstreamRemoteAddress(ConnectionManagerUtility::mutateRequestHeaders(
         *request_headers_, connection_manager_.read_callbacks_->connection(),
         connection_manager_.config_, *snapped_route_config_, connection_manager_.random_generator_,
-        connection_manager_.runtime_, connection_manager_.local_info_));
```

Comment:
Sorry, a question, so we don't use this `connection_manager_.runtime_` anymore?

Reply:
The `runtime_` was only passed through to the `mutateTracingRequestHeader` method, which has now been pulled out and performed after the cached route has been determined.